src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -1,76 +1,57 @@
-import PropTypes from 'prop-types';
+import _ from 'lodash';
```

Comment:
> Try to remove lodash and underscore usage, instead rely on native JS methods

> I think we cannot remove the lodash in this file

Why not? @DylanDylann 

- `_.isEmpty -> isEmptyObject`
- `_.chain -> native methods`
- `_.map -> native map`

Reply:
Updated